Makefile

```diff
@@ -88,12 +88,13 @@ install: manifests
 deploy: install-kustomize
 	kubectl apply -f config/crds
 	kubectl apply -f config/manager/namespace.yaml
-	./scripts/image_patch.sh ./config/default/manager_image_patch.yaml ${IMG}
+	./scripts/image_patch.sh ./config/custom/manager_image_patch.yaml ${IMG}
 	bin/kustomize build $(KUSTOMIZE_BASE) | kubectl apply -f -
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests:
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go all
+	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go rbac --output-dir config/base/rbac
+	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd --output-dir config/base/crds
```

Comment:
I wasn’t able to use https://github.com/kubernetes-sigs/controller-tools/blob/master/cmd/controller-gen/main.go#L207-L208
as it belongs to v0.2.0 with some breaking changes. 
Instead I used `--output-dir` to achieve the same goal; but I had to use separate commands instead of `all`

Reply:
I see, this looks fine to me, thanks!